src/main/java/seedu/address/logic/commands/UndoCommand.java

```diff
@@ -52,6 +54,8 @@ public CommandResult execute(Model model) throws CommandException {
         requireNonNull(model);
 
         if (commandHistory.empty()) {
+            Logger julLogger = LogsCenter.getLogger(UndoCommand.class);
+            julLogger.info(MESSAGE_NO_COMMAND_TO_UNDO);
```

Comment:
What does `julLogger` stand for? Will it be better to use a more descriptive name?

Reply:
Thanks for your kind suggestion.